fission-cli/library/Fission/CLI/Types.hs

```diff
@@ -519,7 +522,7 @@ instance
            | cmd == Just "pin" || cmd == Just "add" -> (Nothing,  opts' <> [arg', timeout, cidVersion, ignore])
            | otherwise                              -> (Nothing,  opts' <> [arg', timeout])
 
-      processStr = intercalate " " ("IPFS_PATH=" <> ipfsRepo : ipfs : opts)
+      processStr = unwords ("IPFS_PATH=" <> ipfsRepo : ipfs : opts)
```

Comment:
I don't always agree with HLint, but I am appreciating having these hints back now that HLS is up and running

Reply:
> now that HLS is up and running
Wait! :star_struck:

Reply:
haha yeah! We were just waiting for HLS 1.0 to hit `<nixpkgs/unstable>`. Once that you upgraded, everything works: inline HLint, hover for type signatures, eta reductions, autocomplete. All the things!

Reply:
and there is **much** rejoicing